collectibles/collectibles.py

```diff
@@ -0,0 +1,223 @@
+        self,
+        ctx: commands.Context,
+        _global: Optional[bool],
+        *,
+        collectible_name: str,
+    ):
+        """Buy a collectible"""
+        _global = True if _global is None else _global
+        maybe_collectible = await self._search_collectibles(
+            ctx, _global, collectible_name  # type:ignore[arg-type]
+        )
+        if maybe_collectible is None:
+            return await ctx.send("I could not find that Collectible")
+        emoji, cost = maybe_collectible
+
+        name = await bank.get_currency_name(ctx.guild)
+        if not await bank.can_spend(ctx.author, cost):
+            bal = await bank.get_balance(ctx.author)
+            return await ctx.send(
+                f"You can't buy that! {collectible_name} costs {cost} {name} but you only have {bal} {name}"
+            )
+        await bank.withdraw_credits(ctx.author, cost)
+        coll_msg = f"You bought a {collectible_name} {emoji} for {cost} {name}"
+        if await ctx.embed_requested():
+            embed = discord.Embed(
+                title="Collectible purchase",
+                colour=await ctx.embed_colour(),
+                description=coll_msg,
+            )
+            embed.timestamp = datetime.utcnow()
+            kwargs = {"embed": embed}
+        else:
+            kwargs = {"content": coll_msg}
+        await ctx.send(**kwargs)
+        async with self.config.user(ctx.author).collectibles() as coll:
+            try:
+                msg = coll[collectible_name]
+            except KeyError:
+                coll[collectible_name] = f"{emoji} 1x"
+            else:
+                emoji, amount = msg.split()
+                amount = int(amount.strip("x"))
+                amount += 1
+                coll[collectible_name] = f"{emoji} {amount}x"
+        func = self.config.user(ctx.author).total_count
+        await func.set(await func() + 1)
+
+    @collectible.command(name="list")
+    async def collectible_list(self, ctx: commands.Context):
+        """List the collectibles available to purchase"""
+        collectibles: dict = {"Global": await self.config.collectibles()}
+        if ctx.guild:
+            collectibles["Guild"] = await self.config.guild(ctx.guild).collectibles()
+        log.debug(collectibles)
+        msg: str = ""
+        for key, value in collectibles.items():
+            log.debug(value)
+            if not value:
+                msg += f"**{key}**\nNo Collectibles available"
+                continue
+            msg += (
+                f"**{key}**\n"
+                + "\n".join(
+                    f"{name} {emoji}: {cost}" for name, (emoji, cost) in value.items()
+                )
+                + "\n\n"
+            )
+        msg = msg[:-2] if msg.endswith("\n") else msg
+        await Menu(source=Page(list(pagify(msg)), "Collectibles")).start(ctx)
+
+    @collectible.command(
+        name="globaladd",
+        aliases=[
+            "gadd",
+        ],
+    )
```

Comment:
```suggestion
    @collectible.command(name="globaladd", aliases=["gadd"])
```
Personally I don't like the styling changes here